src/primitives.rs

```diff
@@ -71,7 +71,7 @@ pub mod bitcoin {
                 bip32::{self, DerivationPath, ExtendedPubKey, Fingerprint},
                 psbt,
             },
-            Address, Network,
+            Address, EcdsaSighashType, Network,
```

Comment:
Are you sure this needs to be part of our domain? Isn’t it an implementation detail of the signer?